### DISCUSSION TITLE
GPU acceleration option while deep learning inference?

### DISCUSSION THREAD

**Discussion by pjw1** [CBICA/CaPTk#1456] [Q&A] [answered]
Hi! 
Thank you for opening your code.

I wonder whether there exists a gpu acceleration option when running deep learning model (e.g. skull stripping in DeepMedic).
I could not find any option related to gpu in DeepMedic, maunal or github issue.

Is there gpu acceleration option while deep learning inference?

Thank you in advance!

**Comment by sarthakpati** [comment, marked as answer]
Hi,

Thank you for your interest in CaPTk!

We currently only support CPUs for inference.

Cheers,
Sarthak
